src/commander/protobuf.h

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Shift Cryptos AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef _COMMANDER_PROTOBUF_H_
+#define _COMMANDER_PROTOBUF_H_
+
+#include <compiler_util.h>
+#include <util.h>
+
+#include <hww.pb.h>
+
+#include <stdbool.h>
+
+USE_RESULT bool protobuf_decode(const in_buffer_t* in_buf, Request* request_out);
+
+void protobuf_encode(const Response* response, buffer_t* buf_out);
```

Comment:
Doc comments please? Especially for encode which aborts on error but it doesn't say so anywhere.

Reply:
done